addon/components/document-view.js

```diff
@@ -271,6 +271,10 @@ export default class DocumentViewComponent extends Component {
         label: "created-by-group",
         sort: true,
       },
+      category: {
+        label: "category",
+        sort: false,
+      },
```

Comment:
i think it would make sense to add a config key to the alexandria config service:
```
documentListColumns: ["...", "...", "category"...]
```
and then cantons can specify which columns and in what order they want to have them.

Reply:
@StephanH90 implemented